packages/lab/src/list-next/ListNext.css

```
@@ -0,0 +1,30 @@
+/* Style applied to the root element */
+.saltListNext {
+  margin: 0;
+  background: var(--salt-container-primary-background);
+  border-color: var(--salt-container-primary-borderColor);
+  border-style: var(--salt-container-borderStyle);
+  border-width: var(--salt-size-border);
+  height: calc(var(--salt-size-stackable) * var(--listNext-displayedItemCount));
+  outline: none;
+  overflow-y: auto;
+  position: relative;
+  user-select: none;
+  padding-inline-start: 0;
+  box-sizing: content-box;
+}
+
+.saltListNext:empty:before {
+  height: 100%;
+  padding-inline: var(--salt-size-unit);
+  display: flex;
+  position: relative;
+  align-items: center;
+  content: var(--listNext-emptyMessage);
+}
+
+/* Pseudo-class applied to the root element on focus */
+.saltListNext-focusable:focus-visible {
+  outline: var(--salt-focused-outline);
+  outline-offset: calc(0 - var(--salt-focused-outlineWidth));
```

```suggestion
  outline-offset: calc(-1 * var(--salt-focused-outlineWidth));
```
easier to understand